Auth0.podspec

```diff
@@ -62,18 +62,18 @@ Pod::Spec.new do |s|
   s.ios.source_files = 'Auth0/*.{swift,h,m}'
   s.ios.frameworks = 'UIKit', 'SafariServices', 'LocalAuthentication'
   s.ios.weak_framework = 'AuthenticationServices'
-  s.ios.dependency 'SimpleKeychain'
+  s.ios.dependency 'SimpleKeychain', '~> 0.11.0'
   s.ios.dependency 'JWTDecode'
   s.osx.source_files = 'Auth0/*.swift'
   s.osx.exclude_files = web_auth_files
-  s.osx.dependency 'SimpleKeychain'
+  s.osx.dependency 'SimpleKeychain', '~> 0.11.0'
   s.osx.dependency 'JWTDecode'
   s.watchos.source_files = 'Auth0/*.swift'
   s.watchos.exclude_files = watchos_exclude_files
   s.watchos.dependency 'JWTDecode'
   s.tvos.source_files = 'Auth0/*.swift'
   s.tvos.exclude_files = web_auth_files
-  s.tvos.dependency 'SimpleKeychain'
+  s.tvos.dependency 'SimpleKeychain', '~> 0.11.0'
```

Comment:
Please leave it unconstrained (also all the instances above).

Reply:
Reverted.